main.py

```diff
@@ -1,7 +1,55 @@
 import init_django_orm  # noqa: F401
 
-from django.db.models import QuerySet
+from db.models import Genre, Actor
 
+genres = [
+    "Western",
+    "Action",
+    "Dramma"
+]
 
-def main() -> QuerySet:
-    pass
+actors = [
+    ("George", "Klooney"),
+    ("Kianu", "Reaves"),
+    ("Scarlett", "Keegan"),
+    ("Will", "Smith"),
+    ("Jaden", "Smith"),
+    ("Scarlett", "Johansson")
+]
+
+
+def main() -> None:
+    for genre in genres:
+        Genre.objects.create(name=genre)
+
+    for actor in actors:
+        for first_name, last_name in actor:
+            Actor.objects.create(
+                first_name=first_name[0],
+                last_name=last_name[1]
+            )
+
+    Genre.objects.filter(
+        name="Dramma"
+    ).update(name="Drama")
+
+    Actor.objects.filter(
+        last_name="Klonney"
+    ).update(last_name="Clooney")
+
+    Actor.objects.filter(
+        first_name="Kianu",
+        last_name="Reaves"
+    ).update(first_name="Keanu", last_name="Reeves")
+
+    Genre.objects.filter(
+        name="Action"
+    ).delete()
+
+    Actor.objects.filter(
+        first_name="Scarlett"
+    ).delete()
+
+    Actor.objects.get(
+        last_name="Smith"
+    )
```

Comment:
Add a return statement with a required queryset as mentioned in the task description.

Reply:
Return:
QuerySet of actors with last_name "Smith" and ordered by first_name

Reply:
fix tests